.github/actions/test-driver/action.yml

```diff
@@ -18,6 +18,8 @@ runs:
     - name: Build static viz frontend
       run: yarn build-static-viz
       shell: bash
+    - name: Setup upterm session
```

Comment:
did you intend for this to be committed?

Reply:
no, I was debugging the failing redshift tests. will remove